daml-assistant/exe/DA/Daml/Assistant.hs

```diff
@@ -302,7 +302,7 @@ withLogger damlPath k = do
     let logOfInterest prio = prio `elem` [L.Telemetry, L.Warning, L.Error]
         gcpConfig = L.GCPConfig
             { gcpConfigTag = "assistant"
-            , gcpConfigDamlPath = Just (unwrapDamlPath damlPath)
+            , gcpConfigCachePath = Just (unwrapDamlPath damlPath)
```

Comment:
This seems incorrect -- you're setting GCP cache path to the DAML home.

Reply:
see my comment above, I think we should choose the default differently.

Reply:
set to `Nothing`, defaults to XDG_CACHE_DIR now.